docs/DeveloperGuide.md

```diff
@@ -249,47 +249,210 @@ Priorities: High (must have) - `* * *`, Medium (nice to have)  - `* *`,  Low (un
 
 
 Priority | As a ... | I want to ... | So that I can...
--------- | :-------- | :--------- | :-----------
+-------- | :---------- | :--------- | :-----------
 `* * *` | new user | see usage instructions | refer to instructions when I forget how to use the App
-`* * *` | user | add a new person |
-`* * *` | user | delete a person | remove entries that I no longer need
-`* * *` | user | find a person by name | locate details of persons without having to go through the entire list
-`* *` | user | hide [private contact details](#private-contact-detail) by default | minimize chance of someone else seeing them by accident
-`*` | user with many persons in the address book | sort persons by name | locate a person easily
+`* * *` | user | add a new task (with deadlines) and events (with start and end timings) | keep track of it and complete it in the future
```

Comment:
how about floating tasks? try to split up the different ttypes - events, tassks, .... I do believe that each different type yields different purposes and it'll be more specific. It is always better to be specific in SE then generalizing things.

e.g. Add tasks with deadlines ... "So that I can be notified/reminded/ .... " (this is just an example)